blackbox/docs/sql/ddl.txt

```diff
@@ -1049,3 +1049,244 @@ means that if you re-create the table using the resulting ``CREATE TABLE``
 statement the settings of the 'old' table may differ from the settings of
 the 'new' table. This is because the table settings are set explicitly on
 creation time.
+
+User defined Functions
+======================
+
+CREATE OR REPLACE
+-----------------
+
+CrateDB supports user-defined functions. See :ref:`ref-create-function`
+for a full syntax description.
+
+These functions can be created, like so::
+
+     cr> CREATE FUNCTION add(integer, integer)
+     ...  RETURNS integer
+     ...  LANGUAGE JAVASCRIPT
+     ...  AS 'function(a, b){
+     ...        return a + b;
+     ...      }';
+
+``OR REPLACE`` can be used to replace an existing function::
+
+     cr> CREATE OR REPLACE FUNCTION add(integer, integer)
+     ...  RETURNS integer
+     ...  LANGUAGE JAVASCRIPT
+     ...  AS 'function(a, b){
+     ...        return a + b;
+     ...      }';
+
+Arguments can be named in the function definition.
+
+For example, if you wanted to name two ``geo_point`` arguments ``start`` and
+``end`` you would do it like so::
+
+     cr> CREATE OR REPLACE FUNCTION distance(start geo_point, end geo_point)
+     ...  RETURNS float
+     ...  LANGUAGE JAVASCRIPT
+     ...  AS 'distance(start_point, end_point){
+     ...        return Math.sqrt( Math.pow(end_point[0] - start_point[0], 2), Math.pow(end_point[1] - start_point[1], 2));
+     ...      }';
+
+.. NOTE::
+
+   Argument names are used for query documentation purposes only. You cannot
+   reference arguments by name in the function body.
+
+Optionally, a schema for the function can be specified. If no schema is
+specified, the function will be created in the current schema.
```

Comment:
- pls point out more what that means in our case. What i'm missing is, that the `schema` is the place where the function is stored and this doesn't mean that the function can be executed only on tables in this schema.
- please also mention that the default schema is used if no schema is provided ( link to the documentation about the default schema. )